csc-site-ssapp/code/pages/dispense-kit.html

```diff
@@ -16,7 +16,7 @@
                       </svg>
           </span>
 
-        <span data-tag='dashboard'>Dashboard</span> / <span data-tag='kits-management'>Kits Management</span> / <span>{{@studyId}} - {{@orderId}}</span>
+        <span data-tag='dashboard'>Dashboard</span> / <span data-tag='kits-management'>Kits Management</span> / <span>{{@studyId}} - {{@orderId}}/ Kits / {{@kitId}} </span>
```

Comment:
where are data-tags for <span>{{@studyId}} - {{@orderId}}/ Kits and {{@kitId}} that assures navigation?